src/wandbot/evaluation/weave_eval/main.py

```diff
@@ -56,6 +66,7 @@ async def get_eval_record(
         "system_prompt": response["system_prompt"],
         "generated_answer": response["answer"],
         "retrieved_contexts": response["source_documents"],
+        "retrieved_contexts_individual": parse_text_to_json(response["source_documents"]),
```

Comment:
remove line 68. no need to add both concatenated and individual contexts.

Reply:
Deleted in the new commit!